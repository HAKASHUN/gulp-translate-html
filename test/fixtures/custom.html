<h1 class="ag-display_n">{{ game.title }}</h1>
<p>{{ game.description }}</p>
