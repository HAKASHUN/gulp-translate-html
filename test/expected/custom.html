<h1 class="ag-display_n">poti</h1>
<p>hello world!</p>
